yaus_c4gt_frontend/src/pages/SignUp.js

```diff
@@ -0,0 +1,341 @@
+import React, { Component, useEffect, useState } from "react";
+import { useHistory } from "react-router-dom";
+
+import {
+  Layout,
+  Menu,
+  Button,
+  Typography,
+  Card,
+  Form,
+  Input,
+  Checkbox,
+} from "antd";
+
+import { Link } from "react-router-dom";
+import {
+  TwitterOutlined,
+  InstagramOutlined,
+  GithubOutlined,
+} from "@ant-design/icons";
+
+const { Title } = Typography;
+const { Header, Footer, Content } = Layout;
+const template = [
```

Comment:
Create the separate assets/icons files for the `svg` variables

Reply:
resolved in f61944537d23eb1850a8461afa4c4c3ef5f8b226